crates/node/src/command.rs

```diff
@@ -265,16 +268,28 @@ pub fn run() -> sc_cli::Result<()> {
                     )?;
                 }
 
-                cli.run.run_cmd.shared_params.chain = Some(madara_path + "/chain-specs/testnet-sharingan-raw.json");
+                cli.run.run_cmd.shared_params.chain =
+                    Some(madara_path.clone() + "/chain-specs/testnet-sharingan-raw.json");
 
                 // This should go apply to all testnets when applying a match pattern
                 cli.run.run_cmd.rpc_external = true;
                 cli.run.run_cmd.rpc_methods = RpcMethods::Unsafe;
             }
 
+            let mut da_config: Option<(DaLayer, PathBuf)> = None;
```

Comment:
`let da_config = if let Some(da_layer) = cli.run.da_layer {} else { None };`
Or `match` if you find it more readable